sql/src/test/java/org/apache/druid/sql/calcite/CalciteQueryTest.java

```diff
@@ -2333,7 +2333,7 @@ public void testExactCountDistinctWithFilter()
   {
     notMsqCompatible();
     final String sqlQuery = "SELECT COUNT(DISTINCT foo.dim1) FILTER(WHERE foo.cnt = 1), SUM(foo.cnt) FROM druid.foo";
-    // When useApproximateCountDistinct=true and useGroupingSetForExactDistinct=false, planning fails due
+    // When useApproximateCountDistinct=false and useGroupingSetForExactDistinct=false, planning fails due
```

Comment:
Could you please explicitly set the param `useApproximateCountDistinct` in the query context below to avoid further confusion and make the test more readable?